zcash_primitives/src/transaction/components/issuance.rs

```diff
@@ -72,7 +70,9 @@ fn read_action<R: Read>(mut reader: R) -> io::Result<IssueAction> {
 
 pub fn read_note<R: Read>(mut reader: R) -> io::Result<Note> {
     let recipient = read_recipient(&mut reader)?;
-    let value = reader.read_u64::<LittleEndian>()?;
+    let mut tmp = [0; 8];
+    reader.read_exact(&mut tmp)?;
+    let value = u64::from_le_bytes(tmp);
```

Comment:
will this work:
`let value = u64::from_le_bytes(reader.read_exact_into_array()?);` ?

Reply:
Nope, need a newer version